sources/packages/backend/apps/api/src/constants/grants.constants.ts

```diff
@@ -0,0 +1,10 @@
+export const CSLF = "CSLF";
```

Comment:
So far we did not have a strong reason to have the grants/loans codes hardcoded. They can be dynamically managed by the workflow new grants can be added without a code change.
Is there a strong reason to have these values hard coded?